ecosystem/sep-0006.md

```diff
@@ -1010,6 +1011,22 @@ Name | Type | Description
 * `too_large` -- deposit/withdrawal size exceeded `max_amount`.
 * `error` -- catch-all for any error not enumerated above.
 
+### Refund Object Schema
+
+Name | Type | Description
+-----|------|------------
+`type` | string | One of `partial` or `full`
+`total_refunded` | string | The total amount refunded to the user, in units of `amount_in_asset`. If `type` is `full`, this amount must match `amount_in`.
```

Comment:
I suggest we use the term amount here so that it is not ambiguous to this being an amount or a count.
```suggestion
`amount_refunded` | string | The total amount refunded to the user, in units of `amount_in_asset`. If `type` is `full`, this amount must match `amount_in`.
```

Reply:
Thats fine by me 👍